__tests__/app.test.js

```diff
@@ -289,3 +287,111 @@ describe("GET /api/articles/:article_id/comments --- error handling", () => {
 // });
 // });
 // });
+
+
+
+describe("PATCH /api/articles/:article_id", () => {
+  const validInput77 = { inc_votes: 77 };
+
+  test("200 status code received when calling api correctly", () => {
+    return request(app).patch("/api/articles/1").send(validInput77).expect(200);
+  });
+
+  test("data received in the format {article: [{}]} with a single object in the array when input body is valid", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send(validInput77)
+      .expect(200)
+      .then((data) => {
+        expect(data.body).toMatchObject({
+          article: expect.any(Object),
+        });
+        expect(Array.isArray(data.body.article)).toBe(true);
+        expect(data.body.article.length).toBe(1);
+      });
+  });
+
+  test("article received is valid article format", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send(validInput77)
+      .expect(200)
+      .then((data) => {
+        expect(data.body.article[0]).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+
+  test("The article's votes is increased by the provided inc_votes in the body of users message", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send(validInput77)
+      .expect(200)
+      .then((data) => {
+        expect(data.body.article[0].votes).toBe(100 + 77); //article 1 in test data is 100 at seed point
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id --- error handling", () => {
+  const validInput88 = { inc_votes: 88 };
+
+  test("400 and {msg: Bad Request - no inc_votes provided} are returned when inc_votes not present", () => {
```

Comment:
This is fine, and doesnt need to be changed, but I might respond with a 200 here but not change any of the data, this would be so that in the future if we wanted to be able to patch something else, it would make our lives more difficult as we would always need to add an `inc_votes` key, and might be a little bit brittle